File: Python3/프로그래머스/3/12938. 최고의 집합/최고의 집합.py
def solution(n, s):
    answer = []
    set_num = 0
    while s > 0:
        std = s // n #기준점이 되는 수
        answer.append(std)
        s -= std
        n -= 1
    return answer if min(answer) > 0 else [-1]

File: C++/프로그래머스/3/12938. 최고의 집합/최고의 집합.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n, int s) {
    vector<int> answer;
    if(s < n) {
        answer.push_back(-1);
    } else {
        int std, set_num;
        while(0 < s) {
            std = s / n--;
            s -= std;
            answer.push_back(std);
        }
    }
    return answer;
}